aws/resource_aws_globalaccelerator_endpoint_group.go

```diff
@@ -0,0 +1,360 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/globalaccelerator"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsGlobalAcceleratorEndpointGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsGlobalAcceleratorEndpointGroupCreate,
+		Read:   resourceAwsGlobalAcceleratorEndpointGroupRead,
+		Update: resourceAwsGlobalAcceleratorEndpointGroupUpdate,
+		Delete: resourceAwsGlobalAcceleratorEndpointGroupDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"listener_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"endpoint_group_region": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"health_check_interval_seconds": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"health_check_path": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
This attribute appears to have a default in the [Global Accelerator API Reference](https://docs.aws.amazon.com/global-accelerator/latest/api/API_CreateEndpointGroup.html), which should be used to perform drift detection:

```suggestion
				Default: "/",
```